### PR TITLE
[Android] Update strings

### PR DESCRIPTION
**Pull request by ar-arvind**
- Fixed incorrect `app name` in some translations.
- Fixed incomplete translation error (Added missing Irish(ga) translation).

### CODE REVIEW

File: Android/app/src/main/res/values-en/strings.xml [hunk 1]
```diff
@@ -1,57 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<!-- This is the app's name, translators please do not translate and put in the
-same text as English version. Developers can consider putting in their own
-translation, this string must be changed before publishing -->
+	<!-- This is the app's name, translators please do not translate and put in the same text as English version. Developers can consider putting in their own translation, this string must be changed before publishing -->
```
Comment (MarvinJWendt) [inline Android/app/src/main/res/values-en/strings.xml]: Hi @ar-arvind, why did you change the comments 🤔 

But all in all the pull looks good to me 👍

Comment (ar-arvind) [reply Android/app/src/main/res/values-en/strings.xml]: Hi @MarvinJWendt, I just reformatted the file to display the comments in single line. Thanks!

**Comment by ar-arvind** [comment]
Hi @simonzhexu, I think `strings.xml file in values-en` is redundant, because the default `strings.xml file in values` is in english language, so we don't need this separate translation file. What do you think about this?

**Comment by simonzhexu** [comment]
@ar-arvind , I agree strings.xml in values-en is redundant, I will remove the file.

**Comment by ar-arvind** [comment]
@simonzhexu 👍
